pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -181,7 +181,7 @@ let
     preferLocalBuild = true;
 
     buildInputs = [makeWrapper];
-    passthru = { unwrapped = neovim; };
+    passthru = { unwrapped = neovim; inherit neovimrcFile; };
```

Comment:
Maybe just call this `configFile` in both cases? I don't feel very strongly about this though.

Reply:
I renamed it to configFile.